pipeline.cpp

```diff
@@ -53,6 +54,16 @@ void processUserInputs(bool & running)
     }
 }
 
+/****************************************
+ * CROSS_PRODUCT
+ * Take two vectors and find their cross 
+ * product.
+ ***************************************/
+float crossProduct(const Vertex & vert1, const Vertex & vert2)
```

Comment:
Minor: You don't really need to pass these by reference if you are also making them const.